environment/k8s_environment.go

```diff
@@ -175,7 +177,7 @@ func (env *k8sEnvironment) GetServiceDetails(remotePort uint16) (*ServiceDetails
 	}
 }
 
-// WriteLogs dumps all the pod logs on test failure
+// WriteLogs dumps all the pod logs within the environment into local log files, used near exclusively on test failure
 func (env *k8sEnvironment) WriteLogs(testLogFolder string) {
```

Comment:
I'd probably rename this to `WriteArtfifacts` or something since it includes a db dump now.